Header header
uint32 week
float64 seconds
float64 latitude
float64 longitude
float64 altitude
float64 ve
float64 vn
float64 vu
float64 pitch
float64 roll
float64 yaw
float64 vc
float64 gyro_x
float64 gyro_y
float64 gyro_z
float64 acc_x
float64 acc_y
float64 acc_z
uint32 NSV1
uint32 NSV2
uint32 age
uint32 align_st
uint32 nav_st
uint32 odo_st
uint32 cs
